docs_outdoornav_user_manual/features/foxglove_visualization.mdx

```diff
@@ -7,8 +7,12 @@ toc_max_heading_level: 4
 ---
 
 We have enabled Foxglove visualization as part of OutdoornNav, for improved debugging and data visualization and analysis.
-To access the Foxglove visualization interface, navigate to the following [URL](http://192.168.131.1/_/foxglove)
-(`platform_pc_ip_address`/\_/foxglove), where the _<platform_pc_ip_address>_ is the IP address of the computer the host (UGV) computer.
+To access the Foxglove visualization interface for standard OutdoorNav configurations, navigate to the following 
+[URL](http://192.168.131.1/_/foxglove) (`platform_pc_ip_address`/\_/foxglove), where the _<platform_pc_ip_address>_ is
+the IP address of the computer the host (UGV) computer.
+
+For Husky Observer platforms specifically, you should access the Foxglove visualization page at http://192.168.132.1/_/foxglove
+when connected to the base station.
```

Comment:
We should confirm with @mhosmar-cpr here, but I thought that all new base stations would use the 192.168.132.x subnet, not just Husky Observer.